docs/admin/addons.rst

```diff
@@ -32,8 +32,8 @@ components.
 
 .. _addon-weblate.cleanup.generic:
 
-Cleanup translation files
--------------------------
+Update PO files to match POT (msgmerge)
+---------------------------------------
```

Comment:
I guess this is an error, this is not a msgmerge.

Reply:
"It is "Update PO files to match POT (msgmerge)" is however what it is currently called in the list of plugins in the web admin UI.
What should it be called? (Will change it in both places)

Reply:
These are different addons. You probably don't see this one as it's not valid for gettext PO files.

Reply:
Ah, nice. Still a bit unclear, but will revert the error.